superset-frontend/src/SqlLab/components/TableElement.jsx

```diff
@@ -207,7 +207,7 @@ class TableElement extends React.PureComponent {
         </div>
         <div className="pull-right">
           {table.isMetadataLoading || table.isExtraMetadataLoading ? (
-            <Loading size={50} position="normal" className="margin-zero" />
```

Comment:
As it was, the size prop passed through to a css property of "size" rather than "height/width" so it never really worked in the first place. Since it's not being used, I'm inclined to scrap it (less code, hooray!) and if we need it for some use case, consider setting up some Theme variables for icon sizes (e.g. s/m/l/xl), and implementing those instead.